src/components/slider/GradientSlider.tsx

```diff
@@ -172,6 +157,5 @@ GradientSlider.displayName = 'GradientSlider';
 GradientSlider.types = GradientSliderTypes;
 // @ts-expect-error
 export default asBaseComponent<GradientSliderProps, ComponentStatics<typeof GradientSlider>>(
```

Comment:
We can add another simplification 
Replace asBaseComponent HOC with the `useThemeProps` (see usages in View component)

Reply:
I got issues with the statics and the forwardRef so I decided to leave it

Reply:
That works for me or at least It didn't throw any errors...
```
export default forwardRef<GradientSliderProps<string | HSLA>, ComponentStatics<typeof GradientSlider>>(GradientSlider);
```

Reply:
Yep, I didn't move the forwardRef... I'll make the change